samples/bot-request-approval/csharp/README.md

```diff
@@ -0,0 +1,113 @@
+---
+page_type: sample
+description: This sample shows feature where user can send task request to his manager and manager can approve/reject request in groupchat.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- csharp
+extensions:
+contentType: samples
+createdDate: "6-12-2021 17:00:25"
+---
+
+# Send task request using universal adaptive cards in groupchat
+
+This sample shows a feature where:
+1. Requester : Can request for any task approval from manager by intiating request in groupchat using bot command `request` and only requester can edit the request card.
+2. Manager : Can see the request raised by user in the same groupchat with an option of approve or reject.
+3. Others: Other members in the group chat can see the request details only.
+
+Requester:
+
+- Initiated request using bot command `request` in groupchat.
+
+  ![Initial Card](BotRequestApproval/Images/InitialCard.png)
+
+- Card will refresh for requester to fill details.
+
+  ![Request Card](BotRequestApproval/Images/RequestCard.png)
+  
+- After submitting requester can edit or cancel the request.
+
+  ![Edit/Cancel Card](BotRequestApproval/Images/EditCancelCard.png)
+
+Manager:
+
+- After user submit the request manager can approve/reject the request.
```

Comment:
Add this line below:
Note: Only the manager of the request will see the buttons to approve/reject the request.

Reply:
added note